api/appsettings.json

```diff
@@ -7,10 +7,13 @@
     }
   },
   "Auth": {
-    "Issuer": "admin", // change url to client and admin
-    "Audience": "http://localhost:5000", // change url to server
+    "Issuer": "http://localhost:4200", // will be removed 
+    "Audience": "http://localhost:5000", // will be removed
     "Secret": "someSecret123---", // better be passed as an enviroment variable
     "TokenLifeTime": "1800" // half of an hour
```

Comment:
Why those will be removed? If they are not used why haven't you removed them already?
You should leave comments only if absolutely necessary
Also JSON doesn't support comments in general